packages/app/services/images.js

```diff
@@ -24,14 +24,15 @@ class ImageServices {
     default_extension = "png",
     checkDb = true
   ) {
-    let domainName = company;
+    let domainName = company.company_name;
     if (checkDb) {
-      const image = await this.imageRepository.fetchImage(company);
+      const image = await this.imageRepository.fetchImage(domainName);
       if (!image) return null;
       domainName = image.company_name;
     }
 
-    const imageUrl = `${default_extension}/${domainName}`;
+    const version = new Date(company.updated_at).getTime();
+    const imageUrl = `${process.env.KEY}/${default_extension}/${domainName}?v=${version}`;
```

Comment:
nit: method functions should be pure and we should probably rely on an external function that reads from environment, that way we can have a logical separation and testable code.

Don't need to fix this here, but something to consider.